cql3/statements/select_statement.hh

```diff
@@ -188,6 +188,7 @@ class indexed_table_select_statement : public select_statement {
     noncopyable_function<dht::partition_range_vector(const query_options&)> _get_partition_ranges_for_posting_list;
     noncopyable_function<query::partition_slice(const query_options&)> _get_partition_slice_for_posting_list;
 public:
+    static constexpr size_t max_base_table_query_result_bytes = 1024*1024;
```

Comment:
We already have a constant for this, namely `query::result_memory_limiter::maximum_result_size`, I think it would be best to use the same constant in all places that mean "max memory normally returned in a page".

Reply:
As Avi already  merged this, please fix in a follow-up.

Reply:
https://github.com/scylladb/scylla/pull/8839